operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/Controller.java

```diff
@@ -5,29 +5,52 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+/**
+ * Interface to be implemented by user-provided controller classes.
+ */
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ElementType.TYPE})
 public @interface Controller {
 
+  /**
+   * String representing no name provided.
```

Comment:
I'd rather use `Constant` than `String` either. People can see the type of the object without having it repeated in the doc.